### PR TITLE
Code review project

### PR DESCRIPTION
**Pull request by Rustamxon7**
### **This PR is created to check the trial project.** 

1. I checked these files and added the README file.
2. Linter folder created for HTML&CSS.
3. Bugs are changed.

### _I would be happy if you can review my code 😊_

### CODE REVIEW

**Comment by Davidosky007** [comment]
Hi @Rustamxon7 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code. :pray: 

 ## Invalid CRR
_Check the comments under the review._

- [ ] You have submitted a CRR without making  a review at all on this project which is a requirement for this project to be reviewed

You should make sure you make reviews at least two, using Github features [Read this link](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/review_your_project.md)
 This Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Rustamxon7 — COMMENTED** [review]
I think your code is fine!

**Review by Rustamxon7 — COMMENTED** [review]
Keep moving!

**Comment by Rustamxon7** [comment]
Try to write your code as clear as possible!

**Review by Rustamxon7 — COMMENTED** [review]
I think it would be better to change your screenshot!

**Review by Rustamxon7 — COMMENTED** [review]
Add value to iframe. 0 value shows an error message.

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello @Rustamxon7 :wave:

_**You've done well so far:**
Your usage of appropriate GitHub flow to set up this project is also commendable :handshake:
Your linter checks are running with no errors :100:
Your Readme looks good :+1:_

__However, you need to make some required changes before moving on to the next assignment.__

> Keep in mind that your long hours of work are appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it :blush: 

### **Required Changes** :recycle: 

_Since this project's goal is to learn how to use GitHub features for reviews and how to communicate in a professional way, the focus will be on how you deliver the review, not if you catch all the bugs._

- [ ] For your PR to be professional, you need to include a descriptive summary about the overall activities you've carried out in this project.
summary


- [ ] Check my comments under your review.

### **Optional Changes** :zap: 
_Every comment with the **OPTIONAL** prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


* * *
**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. You can mention me (@teekaytech) so I can be notified about your comments and respond._

*REMINDER: As described in the [Code reviews limit Policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).*

_**Please make the required changes and submit another review request when you are done.**_

* * *

_Reviewed by:_ [**Taofeek Olalere**](https://github.com/teekaytech)
__Cheers & Happy coding :smiley:__

File: index.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
```
[the remark this answers is not in the document]
Comment (Rustamxon7) [reply index.html:1]: I appreciate your help! Thanks, I will try to correct them all step by step.

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+iframe {
+  border: 5px;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  font-size: 15px;
+  font-family: sans-serif;
```
Comment (Rustamxon7) [inline styles.css:13]: Looks great to me and my suggestion is to try to add Google fonts to your code.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+<html lang="en">
```
Comment (Rustamxon7) [inline index.html:2]: Your HTML file looks fine!

File: styles.css [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+  line-height: 1.4;
```
Comment (Rustamxon7) [inline styles.css:14]: CSS file works good.

**Review by Rustamxon7 — COMMENTED** [review]
# Changes not required 

Please check my comments below. 

Submit for a re-review when you're completed

   Happy coding! 👨‍💻
   Keep move 💪💪💪  

- [ ]

**Review by smcommits — CHANGES_REQUESTED** [review]
Hi @Rustamxon7, 

### Great work on making the changes requested by a previous reviewer 👏 
### You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## Required Changes ♻️

- [x] **As the previous reviewer mentioned, it is very important to provide descriptive reviews. You should always include "why" of anything. It will justify your request and will help the audience to better understand the requested change. Please add more description to your comments.**

For instance, you mentioned that "My suggestion is to add Google fonts to your code". This is a great suggestion, but it is not very descriptive. Why should one use google fonts? How will it help in making the codebase better? What will be the overall benefits? 

Please refer to the following great explanation from the previous reviewer. 

```
It is better to always include an explanation, a "why", for each requested change.
For example:

This is a well-explained request: Why don't you use an article tag for the .first-article element instead of a div, that would improve the HTML structure and make the page more accessible
 
 This request lacks a "why": Why don't you use an article tag here instead of a div?

```
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

---
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: styles.css [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+
+#nav-bar a {
+  color: white;
+  font-size: 24px;
+  font-weight: bold;
+  text-decoration: none;
+  margin-right: 1rem;
+}
+
+.vid {
```
Comment (Rustamxon7) [inline styles.css:41]: Why should one use a clear class name? How will it help in making the codebase more understandable?

File: index.html [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
+    <header id="header">
+      <img
+        id="header-img"
+        src="https://www.pngfind.com/pngs/m/180-1800308_twitch-logo-png-transparent-background-twitch-logo-no.png"
+        alt="logo"
+      />
```
Comment (Rustamxon7) [inline index.html:17]: Why not try using a real icon instead of img? This will make your website look more beautiful.

File: styles.css [hunk 7]
```diff
@@ -0,0 +1,69 @@
+* {
```
Comment (Rustamxon7) [inline styles.css:1]: Why not try adding a media query to your CSS file? It helps you make your website responsive.

**Review by Rustamxon7 — COMMENTED** [review]
### **Changes required♻️**

Please check my comments below and make all required changes. 

Submit for a re-review when you completed it.

### Happy Coding 💪💪💪🦾

**Review by smcommits — APPROVED** [review]
### Status: Approved 👍🏽 

Hi @Rustamxon7, 

## You have done a great job! Your project is complete. This PR is now approved. You can go ahead and merge the branch. Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Comment by Rustamxon7** [comment]
Thank you for reviewing my code!

**Comment by Rustamxon7** [comment]
Did I really done it?

**Comment by Rustamxon7** [comment]
> ### Status: Approved 👍🏽
> Hi @Rustamxon7,
> 
> ## You have done a great job! Your project is complete. This PR is now approved. You can go ahead and merge the branch. Congratulations! 🎉
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Did I really do it?

**Comment by smcommits** [comment]
> > ### Status: Approved 👍🏽
> > Hi @Rustamxon7,
> > ## You have done a great job! Your project is complete. This PR is now approved. You can go ahead and merge the branch. Congratulations! tada
> > Cheers and Happy coding!clapclapclap
> > Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
> 
> Did I really do it?

@Rustamxon7 Absolutely you did! You added a "why" to every request. It made your request more descriptive. You have done a great job!

**Pull request merged**